sdk/java/dagger-codegen-maven-plugin/src/main/java/io/dagger/codegen/introspection/ObjectVisitor.java

```diff
@@ -225,30 +225,34 @@ private void buildFieldMethod(
       fieldMethodBuilder
           .addException(InterruptedException.class)
           .addException(ExecutionException.class)
-          .addException(ClassName.bestGuess("DaggerQueryException"));
+          .addException(ClassName.get("io.dagger.client.exception", "DaggerQueryException"))
+          .addException(ClassName.get("io.dagger.client.exception", "DaggerExecException"));
```

Comment:
Just wondering if we need both explicitly or the `DaggerQueryException` should cover them

Reply:
yes it was enough! i have a doubt about that because i think there was a specific catch somewhere